src/theme/dark.ts

```diff
@@ -307,7 +307,11 @@ const controlBar = {
   shadow: shadows.large,
   bgd: colors.greys.grey100,
   border: 'none',
-  opacity: 1
+  opacity: 1,
+  selected: {
+    text: buttons.primary.active.text,
+    bgd: buttons.primary.active.bgd,
```

Comment:
selected?

Reply:
That would help, wouldn't it? Updating.

Reply:
I do that all the time. It's specifically hard to compared updates here vs crux. I'm still getting used to it.